openapi/paths/auth/refresh-token.yml

```diff
@@ -0,0 +1,68 @@
+post:
+  operationId: "RefreshToken"
+  parameters:
+    - in: header
+      name: refreshToken
+      required: true
+      schema:
+        type: string
+  responses:
+    "200":
+      description: "200 response"
+      headers:
+        Access-Control-Allow-Origin:
+          schema:
+            type: "string"
+        Access-Control-Allow-Methods:
+          schema:
+            type: "string"
+        Access-Control-Allow-Headers:
+          schema:
+            type: "string"
+      content: {}
+  x-amazon-apigateway-request-validator: "ValidateBodyAndQuery"
+  x-amazon-apigateway-integration:
+    type: "aws"
+    httpMethod: "POST"
+    uri: "arn:aws:apigateway:<<AWS_REGION>>:lambda:path/2015-03-31/functions/arn:aws:lambda:<<AWS_REGION>>:<<ACCOUNT_ID>>:function:<<ENVIRONMENT>>-refresh-token-lambda/invocations"
```

Comment:
Seems we can not publish API specs with filling up AWS information which is sensitive 🤔

Reply:
yes. looking into it.